conans/client/command.py

```diff
@@ -71,6 +72,18 @@ def __call__(self, parser, namespace, values, option_string=None):
         setattr(namespace, self.dest, values)
 
 
+class ConanFilePath(argparse.Action):
```

Comment:
Probably this should be used in other commands like install, create ... Create an engineering issue

Reply:
Done, it will require more triaging, but we won't forget about it: https://github.com/conan-io/conan/issues/4157